src/scan_nut.cc

```diff
@@ -134,6 +134,71 @@ s_nut_dumpdata_to_fty_message (fty_proto_t *fmsg, std::map <std::string, std::st
     }
 }
 
+void
+s_nut_dumpdata_daisychain_to_fty_message (fty_proto_t *asset, std::map <std::string, std::string> &dump, zsock_t* pipe)
+{
+    if(! asset) return;
+
+    auto item = dump.find ("device.count");
+    if(item == dump.end () || streq(item->second.c_str(), "1")) {
```

Comment:
To be on the safe and sane side, this should check that the returned value is a number (integer), and is `>1` to get to the next part; for `<=1` it should return as done here.